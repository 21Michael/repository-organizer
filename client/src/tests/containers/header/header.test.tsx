
import React from "react";
import { render, cleanup } from "@testing-library/react";
import Header from "../../../containers/header/header";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("Header: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("header", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Header />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
