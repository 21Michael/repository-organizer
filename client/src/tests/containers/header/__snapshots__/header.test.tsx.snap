// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header:  header 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="header__wrapper"
    >
      <nav
        class="nav"
      >
        <ul
          class="list"
        >
          <li
            class="item"
          >
            <a
              aria-current="page"
              class="link active"
              href="/"
            >
              repository
            </a>
          </li>
          <li
            class="item"
          >
            <a
              class="link"
              href="/note"
            >
              note
            </a>
          </li>
        </ul>
      </nav>
      <div
        class="auth"
      >
        <a
          class="link"
          href="/signin"
        >
          login
        </a>
      </div>
    </div>
  </header>
</div>
`;
